### ISSUE TITLE
UIPopoverPresentationController should have a non-nil sourceView or barButtonItem set before the presentation occurs.

### ISSUE THREAD

**Issue by MathieuAlphamosa** [qonversion/flutter-sdk#230]
Hello!
I'm having the following error popping up from iPad users on my app (via sentry.io)
```NSGenericException: UIPopoverPresentationController (<UIPopoverPresentationController: 0x103f0d320>) should have a non-nil sourceView or barButtonItem set before the presentation occurs.```

It seems to be related to this: https://stackoverflow.com/a/25644145

**Comment by suriksarkisyan** [comment]
Hi @MathieuAlphamosa 
Could you please explain why do you think this issue is related to our SDK? Which part of our SDK do you use before getting this error?

**Comment by MathieuAlphamosa** [comment]
I'm using the popover presentation style.
```
    var config = new QScreenPresentationConfig(QScreenPresentationStyle.popover, true);
    var config = new QScreenPresentationConfig(QScreenPresentationStyle.push, true);
    Automations.getSharedInstance().setScreenPresentationConfig(config);
```
I have the error when the user display this screen.

**Comment by suriksarkisyan** [comment]
Hello everyone. I apologize for the late response and fix.

The situation is that the presentation style of the popover on iPad differs from how it works on iOS. I have fixed the crash, and now everything works correctly. One small complication is that from cross-platform frameworks (whether Flutter or any other), it is not possible to pass a custom view through the delegate from which this popover will be opened. So for now, everything works correctly, but it is not possible to customize the opening as we can do in our native iOS SDK.

In any case, the bug has been fixed—thank you for the issue!
